src/foam/nanos/auth/Group.js

```diff
@@ -122,6 +122,11 @@ foam.CLASS({
         class: 'foam.u2.view.FObjectPropertyView',
         readView: { class: 'foam.u2.detail.VerticalDetailView' }
       }
+    },
+    {
+      class: 'Boolean',
+      name: 'template',
```

Comment:
@zahnno this only works for this specific case.  To be honest, I'd rather hard coded groups for this ablii access thing, rather than this.   There would be no way to distinguish which group of templates were to be used together. 
The idea of access levels for abliii apps should be DAO based or perhaps we can use the maker/checker stuff that liquid is using. 
And each ablii app may have some app specific hierarchy.  So this is specific to payables for example.